src/java.base/share/classes/sun/nio/fs/PollingWatchService.java

```diff
@@ -61,6 +61,11 @@
 class PollingWatchService
     extends AbstractWatchService
 {
+    // Wait between polling thread creation and first poll (seconds)
+    private static final int POLLING_INIT_DELAY = 1;
+    // Default time between polls (seconds)
```

Comment:
Thanks for more the constant. You can insert a blank line between them to make it easier to distinguish and also to keep it consistent with the existing code.